slices/libsubid4.yaml

```diff
@@ -0,0 +1,16 @@
+package: libsubid4
+
+esential:
+  - libsubid4_copyright
+
+slices:
+  libs:
+    essential:
+      - libaudit1_libs
+      - libc6_libs
+      - libselinux1_libs
```

Comment:
missing dependencies:
- libcrypt1
- libpam0g
- libsemanage2

Reply:
Here I get only these: `Depends: libaudit1 (>= 1:2.2.1), libc6 (>= 2.38), libselinux1 (>= 3.1~)`

Reply:
Those extra libraries are needed for armhf, i386 and riscv64 architectures.